docs/configuration.md

```diff
@@ -92,6 +92,10 @@ Generic placeholders are defined as follows:
 
   # The markdown content to be displayed on the UI home page
   [ information: <string> ]
+
+  # If provided, Perses server will look to the different folders configured and populate the database based on what it is found
+  # The data coming from the provisioning folder will totally override what exists in the database.
```

Comment:
When you say `The data coming from the provisioning folder will totally override what exists in the database.`, you mean it will delete the existing data and than insert the new one?

Reply:
actually it updates the existing data. But it will replace whatever you change in the `spec`